apps/barista-design-system/src/pages/decision-graph-page/components/ba-decision-graph/ba-decision-graph-node/ba-decision-graph-node.spec.ts

```diff
@@ -119,11 +115,7 @@ describe('BaDecisionGraphNode', () => {
     component.node = wrongIdNode;
     fixture.detectChanges();
     expect(component._decisionGraphSteps.length).toBe(1);
-    dispatchFakeEvent(
-      fixture.debugElement.query(By.css('.ba-uxd-edge-button')).nativeElement,
-      'click',
-    );
-    expect(component._decisionGraphSteps.length).toBe(1);
+    expect(() => component.setNextNode(component.node.path[0])).toThrowError();
   });
 
   it('should set started to false when steps array is lower than 2', () => {
```

Comment:
Can we add a test that will check if the error is actually thrown?

Reply:
Did you mean L118?
Well I am testing if the error is thrown
`expect(() => component.setNextNode(component.node.path[0])).toThrowError();`

Reply:
yeah exactly. Terribly sorry that I missed that